File: features/ondemand/klivekit/src/main/java/org/intelehealth/klivekit/model/ChatResponse.java
package org.intelehealth.klivekit.model;

import java.util.ArrayList;

/**
 * Created by Vaghela Mithun R. on 24-07-2023 - 23:23.
 * Email : mithun@intelehealth.org
 * Mob   : +919727206702
 **/
public class ChatResponse {
    private boolean success;
    private ArrayList<ChatMessage> data;

    public ArrayList<ChatMessage> getData() {
        return data;
    }

    public boolean isSuccess() {
        return success;
    }
}

File: klivekit/src/main/java/org/intelehealth/klivekit/restapi/response/ChatResponse.kt
package org.intelehealth.klivekit.restapi.response

import org.intelehealth.klivekit.model.ChatMessage

/**
 * Created by Vaghela Mithun R. on 30-08-2023 - 15:39.
 * Email : mithun@intelehealth.org
 * Mob   : +919727206702
 **/
class ChatResponse<T>(
    var success: Boolean = false,
    val data: T? = null
)